com.unity.render-pipelines.high-definition/Documentation~/Default-Settings-Window.md

```diff
@@ -9,50 +9,50 @@ The High Definition Render Pipeline (HDRP) adds the HDRP Settings tab to Unity's
 | Light Layer Names                     | The name displayed on Lights and Meshes when using [Light Layers](Light-Layers.md). |
 | Decal Layer Names                     | The name displayed on decals and Meshes when using [Decal Layers](Decal.md). |
 
-### Custom Post Processes Order
+## Custom Post Process Orders
 
 Use this section to select which custom post processing effect will be used in the project and in which order they will be executed.
 You have one list per post processing injection point: `After Opaque And Sky`, `Before Post Process` and `After Post Process`. See the [Custom Post Process](Custom-Post-Process.md) documentation for more details.
 
-### Diffusion Profile List
-
-Use this section to select which custom [Diffusion Profiles](Diffusion-Profile.md) can be in view at the same time. To use more than 15 custom Diffusion Profiles in a Scene, use the [Diffusion Profile Override](Override-Diffusion-Profile.md) inside a Volume. This allows you to specify which Diffusion Profiles to use in a certain area (or in the Scene if the Volume is global).
-
-### Miscellaneous
+## Miscellaneous
 
 | **Property**              | **Description**                                              |
 | --------------------------| ------------------------------------------------------------ |
 | Shader Variant Log Level              | Use the drop-down to select what information HDRP logs about Shader variants when you build your Unity Project. • Disabled: HDRP doesn’t log any Shader variant information.• Only HDRP Shaders: Only log Shader variant information for HDRP Shaders.• All Shaders: Log Shader variant information for every Shader type. |
 | Lens Attenuation Mode                 | Set the attenuation mode of the lens that is used to compute exposure. With imperfect lens some energy is lost when converting from EV100 to the exposure multiplier, while a perfect lens has no attenuation and no energy is lost. |
 | Use DLSS Custom Project Id            | Controls whether to use a custom project ID for the NVIDIA Deep Learning Super Sampling module. If you enable this property, you can use **DLSS Custom Project Id** to specify a custom project ID.<br/>This property only appears if you enable the NVIDIA package (com.unity.modules.nvidia) in your Unity project. |
 | DLSS Custom Project Id                | Controls whether to use a custom project ID for the NVIDIA Deep Learning Super Sampling (DLSS) module. If you enable this property, you can use **DLSS Custom Project Id** to specify a custom project ID. If you disable this property, Unity generates a unique project ID. <br/>This property only appears if you enable the NVIDIA package (com.unity.modules.nvidia) in your Unity project. |
+| Runtime Debug Shaders                 | Disable this property to remove all debug display shader variants when you build for the Unity Player. This decreases build time but as a result, you cannot use Rendering Debugger in Player builds. |
```

Comment:
Checking my understanding: 
* You can only use the Render Pipeline Debugger window with development builds if this setting is enabled
* It works by generating "debug display" shader variants

Questions:
* Which shaders generate debug display variants: the standard HDRP prebuilt ones/all shaders created using Shader Graph/something else? 
* What do the debug display variants do - e.g. enable special debug views like wireframe?
* Does a user have to do anything else to generate these variants?
* What happens in a development build if you don't generate these variants - you can't use it at all? Some modes don't work properly?

Reply:
Your understanding is correct. 
- The DEBUG_DISPLAY variants are generated by HDRP prebuilt and shader graph shaders. @JulienIgnace-Unity knows more about details related to HDRP.
- Things like wireframe, viewing material channels (albedo, specular etc.), lighting debug etc. Almost all debug options in Rendering Debugger Window work by activating these variants.
- If DEBUG_DISPLAY variants are not there, the debug modes do not work (renders as normal / white / black / pink - not sure actually). This PR doesn't affect this behavior.